packages/lesswrong/lib/collections/petrovDayActions/collection.ts

```diff
@@ -0,0 +1,13 @@
+import schema from './schema';
+import { createCollection } from '../../vulcan-lib';
+import { addUniversalFields } from '../../collectionUtils'
+
+export const PetrovDayActions: PetrovDayActionsCollection = createCollection({
+  collectionName: 'PetrovDayActions',
+  typeName: 'PetrovDayAction',
+  schema
+});
+
+addUniversalFields({collection: PetrovDayActions})
+
+export default PetrovDayActions;
```

Comment:
We should probably have a `checkAccess` which forbids reads at the collection level unless you own the item (`userOwns`) or are an admin, in addition to gating reads at the field level.